README.md

```diff
@@ -9,6 +9,9 @@ Getting started
 To deploy a new BeeKeeper instance, clone this repo, and then run::
 
     $ heroku create
+    $ heroku config:set DISABLE_COLLECTSTATIC=1
+    # Follow the SECRET_KEY instructions below and
+    $ heroku config:set SECRET_KEY=<your secret key>
```

Comment:
Realistically I think we should set up a heroku button (https://devcenter.heroku.com/articles/heroku-button) and create an `app.json` that does all this for us. You can specify that heroku should create a random secret key and shove it in the `env` in the `app.json`